### PR TITLE
New Indonesian Translation: Editor Setup page

### LINKED ISSUE #395: Editor Setup

### PR DESCRIPTION
**Pull request by agprst**
Hi, this is a PR for https://github.com/reactjs/id.react.dev/issues/330 
Translation for Editor Setup page under the Installation section

Closes #395

### CODE REVIEW

File: src/content/learn/editor-setup.md [hunk 1]
```diff
@@ -1,62 +1,62 @@
...
-A properly configured editor can make code clearer to read and faster to write. It can even help you catch bugs as you write them! If this is your first time setting up an editor or you're looking to tune up your current editor, we have a few recommendations.
+Sebuah editor yang terkonfigurasi dengan baik dapat membuat kode lebih mudah dibaca dan lebih cepat ditulis. Ini bahkan dapat membantu Anda menangkap *bug* saat Anda menulisnya! Jika ini adalah pertama kalinya Anda mempersiapkan editor atau Anda ingin menyetel ulang editor Anda saat ini, kami memiliki beberapa rekomendasi.
 
 </Intro>
 
 <YouWillLearn>
 
-* What the most popular editors are
-* How to format your code automatically
+* Apa saja editor yang populer
```
Comment (fzn0x) [inline src/content/learn/editor-setup.md:13]: Apa saja editor yang paling populer

**Review by fzn0x — COMMENTED** [review]
Sedikit improvisasi pada kalimat

File: src/content/learn/editor-setup.md (hunk 1 above)
Comment (agprst) [reply src/content/learn/editor-setup.md]: done, thanks for the suggestion!

File: src/content/learn/editor-setup.md [hunk 2]
```diff
@@ -1,62 +1,62 @@
...
 
-* What the most popular editors are
-* How to format your code automatically
+* Apa saja editor yang paling populer
+* Cara memformat kode Anda secara otomatis
 
 </YouWillLearn>
 
-## Your editor {/*your-editor*/}
+## Editor Anda {/*your-editor*/}
```
Comment (mazipan) [inline src/content/learn/editor-setup.md:18]: Ini slug nya perlu diganti juga gak sih?

File: src/content/learn/editor-setup.md [hunk 3]
```diff
@@ -1,62 +1,62 @@
...
+* [WebStorm](https://www.jetbrains.com/webstorm/) adalah lingkungan pengembangan terpadu yang dirancang khusus untuk JavaScript.
+* [Sublime Text](https://www.sublimetext.com/) memiliki dukungan untuk JSX dan TypeScript, [*syntax highlighting*](https://stackoverflow.com/a/70960574/458193) dan penyelesaian otomatis yang sudah *built-in*.
+* [Vim](https://www.vim.org/) adalah editor teks yang sangat dapat dikonfigurasi yang dirancang untuk membuat membuat dan mengubah jenis teks apa pun menjadi sangat efisien. Ini disertakan sebagai "vi" dengan sebagian besar sistem UNIX dan dengan Apple OS X.
 
-## Recommended text editor features {/*recommended-text-editor-features*/}
+## Fitur editor teks yang direkomendasikan {/*recommended-text-editor-features*/}
 
-Some editors come with these features built in, but others might require adding an extension. Check to see what support your editor of choice provides to be sure!
+Beberapa editor dilengkapi dengan fitur-fitur ini secara *built-in*, tetapi yang lain mungkin memerlukan penambahan ekstensi. Periksa untuk melihat dukungan yang disediakan editor pilihan Anda untuk memastikan!
```
Comment (mazipan) [inline src/content/learn/editor-setup.md:30]:
```suggestion
Beberapa editor memiliki fitur-fitur ini secara *built-in*, tetapi yang lain mungkin memerlukan penambahan ekstensi. Periksa untuk melihat dukungan yang disediakan editor pilihan Anda untuk memastikan!
```

File: src/content/learn/editor-setup.md [hunk 4]
```diff
@@ -1,62 +1,62 @@
...
 
-### Linting {/*linting*/}
+### Pemeriksaan {/*linting*/}
```
Comment (mazipan) [inline src/content/learn/editor-setup.md:32]: Karena *Linting* ini kan agak2 refer ke istilah teknis, bagaimana kalau kita tetap mempertahankan kata-kata *Linting* nya juga?

File: src/content/learn/editor-setup.md (hunk 2 above)
Comment (agprst) [reply src/content/learn/editor-setup.md:18]: according to the [guidelines](https://github.com/reactjs/id.react.dev/blob/main/TRANSLATION.md), translating the Heading ID will ruin the navigation

**Review by mazipan — APPROVED** [review]
Overall udah cukup mudah dipahami menurutku, tinggal nunggu sedikit perbaikan bisa di*merge* ke `main` nih.

File: src/content/learn/editor-setup.md (hunk 3 above)
Comment (mazipan) [reply src/content/learn/editor-setup.md:30]: Ini ngubah kata-kata aslinya, tapi menurutku maksudnya lebih kena

Comment (agprst) [reply src/content/learn/editor-setup.md]: thanks for the suggestion!

File: src/content/learn/editor-setup.md (hunk 4 above)
Comment (mazipan) [reply src/content/learn/editor-setup.md:32]: Gua sebagai orang teknis pun bisa kehilangan maksud ketika "Linting" langsung ditranslasi pake kata "Pemeriksaan".

Comment (agprst) [reply src/content/learn/editor-setup.md:32]: yeah, that's what I thought. I can't find the word in the [glossary](https://github.com/reactjs/id.react.dev/wiki/Glosarium)

Comment (agprst) [reply src/content/learn/editor-setup.md]: done, thanks for the suggestion!

Changes:
Pemeriksaan -> *Linting*
Pemeriksa -> *Linter*

**Review by mazipan — APPROVED** [review]

**Comment by fzn0x** [comment]
Sepertinya install disini tidak mengikuti glossary, install seharusnya menjadi `instal`

Need to confirm @resir014

File: src/content/learn/editor-setup.md [hunk 5]
```diff
@@ -1,62 +1,62 @@
...
 
-### Linting {/*linting*/}
+### *Linting* {/*linting*/}
 
-Code linters find problems in your code as you write, helping you fix them early. [ESLint](https://eslint.org/) is a popular, open source linter for JavaScript. 
+*Linter* kode dapat menemukan masalah dalam kode Anda saat Anda menulis, membantu Anda memperbaikinya lebih awal. [ESLint](https://eslint.org/) adalah *linter open source* yang populer untuk JavaScript.
 
-* [Install ESLint with the recommended configuration for React](https://www.npmjs.com/package/eslint-config-react-app) (be sure you have [Node installed!](https://nodejs.org/en/download/current/))
-* [Integrate ESLint in VSCode with the official extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)
+* [Memasang ESLint dengan konfigurasi yang direkomendasikan untuk React](https://www.npmjs.com/package/eslint-config-react-app) (pastikan Anda telah [memasang Node!](https://nodejs.org/en/download/current/))
```
Comment (resir014) [inline src/content/learn/editor-setup.md:36]:
```suggestion
* [Menginstal ESLint dengan konfigurasi yang direkomendasikan untuk React](https://www.npmjs.com/package/eslint-config-react-app) (pastikan Anda telah [menginstal Node!](https://nodejs.org/en/download/current/))
```

`install` -> `instal`, as per the [Glossary](https://github.com/reactjs/id.react.dev/wiki/Glosarium)

File: src/content/learn/editor-setup.md [hunk 6]
```diff
@@ -1,62 +1,62 @@
...
+Hal terakhir yang ingin Anda lakukan saat berbagi kode Anda dengan kontributor lain adalah terlibat dalam diskusi tentang [tab vs spasi](https://www.google.com/search?q=tabs+vs+spaces)! Untungnya, [Prettier](https://prettier.io/) akan membersihkan kode Anda dengan memformat ulang sesuai dengan aturan yang telah ditentukan. Jalankan Prettier, dan semua tab Anda akan dikonversi menjadi spasi—dan indentasi, tanda kutip, dll juga akan diubah sesuai dengan konfigurasi. Dalam pengaturan yang ideal, Prettier akan berjalan saat Anda menyimpan file Anda, dengan cepat membuat perubahan ini untuk Anda.
 
-You can install the [Prettier extension in VSCode](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) by following these steps:
+Anda dapat memasang [ekstensi Prettier di VSCode](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) dengan mengikuti langkah-langkah berikut:
```
Comment (resir014) [inline src/content/learn/editor-setup.md:45]:
```suggestion
Anda dapat menginstal [ekstensi Prettier di VSCode](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) dengan mengikuti langkah-langkah berikut:
```

**Review by resir014 — CHANGES_REQUESTED** [review]

File: src/content/learn/editor-setup.md [hunk 7]
```diff
@@ -1,62 +1,62 @@
...
+* [Vim](https://www.vim.org/) adalah editor teks yang sangat dapat dikonfigurasi yang dirancang untuk membuat membuat dan mengubah jenis teks apa pun menjadi sangat efisien. Ini disertakan sebagai "vi" dengan sebagian besar sistem UNIX dan dengan Apple OS X.
 
-## Recommended text editor features {/*recommended-text-editor-features*/}
+## Fitur editor teks yang direkomendasikan {/*recommended-text-editor-features*/}
 
-Some editors come with these features built in, but others might require adding an extension. Check to see what support your editor of choice provides to be sure!
+Beberapa editor telah memiliki fitur-fitur ini secara *built-in*, tetapi yang lain mungkin memerlukan penambahan ekstensi. Periksa untuk melihat dukungan yang disediakan editor pilihan Anda untuk memastikan!
 
-### Linting {/*linting*/}
+### *Linting* {/*linting*/}
```
Comment (resir014) [inline src/content/learn/editor-setup.md:32]:
```suggestion
### Linting {/*linting*/}
```

We suggest avoiding italics in heading text.

We should also probably add this to the Style Guide @mhaidarhanif @r17nx @zainfathoni

Comment (agprst) [reply src/content/learn/editor-setup.md]: thanks for the suggestion!

Comment (mhaidarhanif) [reply src/content/learn/editor-setup.md]: Yes, should be on the Style Guide then.

**Comment by mazipan** [comment]
Waiting your approval @resir014

File: src/content/learn/editor-setup.md [hunk 8]
```diff
@@ -1,62 +1,62 @@
...
+2. Ketik "settings"
+3. Tekan Enter
+4. Di bilah pencarian, ketik "format on save"
+5. Pastikan opsi "format on save" dicentang!
 
-> If your ESLint preset has formatting rules, they may conflict with Prettier. We recommend disabling all formatting rules in your ESLint preset using [`eslint-config-prettier`](https://github.com/prettier/eslint-config-prettier) so that ESLint is *only* used for catching logical mistakes. If you want to enforce that files are formatted before a pull request is merged, use [`prettier --check`](https://prettier.io/docs/en/cli.html#--check) for your continuous integration.
+Jika *preset* ESLint Anda memiliki aturan pemformatan, mereka mungkin akan bertentangan dengan Prettier. Kami menyarankan untuk menonaktifkan semua aturan pemformatan di *preset* ESLint Anda menggunakan [`eslint-config-prettier`](https://github.com/prettier/eslint-config-prettier) sehingga ESLint hanya digunakan untuk menangkap kesalahan logis. Jika Anda ingin menegakkan bahwa file diformat terlabih dahulu sebelum *pull request* digabungkan, gunakan [`prettier --check`](https://prettier.io/docs/en/cli.html#--check) untuk integrasi berkelanjutan Anda.
```
Comment (r17x) [inline src/content/learn/editor-setup.md:62]: * Typo
* Italic for `file`
```suggestion
Jika *preset* ESLint Anda memiliki aturan pemformatan, mereka mungkin akan bertentangan dengan Prettier. Kami menyarankan untuk menonaktifkan semua aturan pemformatan di *preset* ESLint Anda menggunakan [`eslint-config-prettier`](https://github.com/prettier/eslint-config-prettier) sehingga ESLint hanya digunakan untuk menangkap kesalahan logis. Jika Anda ingin menegakkan bahwa *file* diformat terlebih dahulu sebelum *pull request* digabungkan, gunakan [`prettier --check`](https://prettier.io/docs/en/cli.html#--check) untuk integrasi berkelanjutan Anda.
```

Comment (r17x) [inline src/content/learn/editor-setup.md:62]: * Typo
* Italic for `file`
```suggestion
Jika *preset* ESLint Anda memiliki aturan pemformatan, mereka mungkin akan bertentangan dengan Prettier. Kami menyarankan untuk menonaktifkan semua aturan pemformatan di *preset* ESLint Anda menggunakan [`eslint-config-prettier`](https://github.com/prettier/eslint-config-prettier) sehingga ESLint hanya digunakan untuk menangkap kesalahan logis. Jika Anda ingin menegakkan bahwa *file* diformat terlebih dahulu sebelum *pull request* digabungkan, gunakan [`prettier --check`](https://prettier.io/docs/en/cli.html#--check) untuk integrasi berkelanjutan Anda.
```

**Review by r17x — CHANGES_REQUESTED** [review]

File: src/content/learn/editor-setup.md (hunk 8 above)
Comment (agprst) [reply src/content/learn/editor-setup.md]: thanks!

**Review by r17x — APPROVED** [review]

**Review by mhaidarhanif — APPROVED** [review]
Waiting for @resir014

**Review by mazipan — APPROVED** [review]

**Review by zainfathoni — APPROVED** [review]

**Comment by mazipan** [comment]
Waiting you, @resir014

**Comment by mazipan** [comment]
Bump @resir014 again

**Review by resir014 — APPROVED** [review]
🎉

**Pull request merged**